### ISSUE TITLE
PossibleUserWarning: Shuffling enabled in 'val_dataloader' during FlowGenerator Training

### ISSUE THREAD

**Issue by AnanyaPassi-prog** [jbohnslav/deepethogram#121]
I started training the FlowGenerator on my videos. The training seems to be going fine - since several epochs are being completed. However, before each epoch I get the following error:

<pre>/home/anuj/miniconda3/envs/deg/lib/python3.7/site-packages/pytorch_lightning/trainer/connectors/data_connector.py:498: PossibleUserWarning: Your `val_dataloader`&apos;s sampler has shuffling enabled, it is strongly recommended that you turn shuffling off for val/test/predict dataloaders.</pre>

I have not made any change to the default codes provided by DeepEthogram yet. Would anyone recommend doing this?

**Comment by giannottigiuseppe** [comment]
Hi, this is just a warning. The training should proceed without any problem!
